src/utils/validation.tsx

```diff
@@ -0,0 +1,25 @@
+type validationProps = {
+  id: string;
+  value: string;
+  isValidInfos: any;
+  setIsValidInfos: any;
+}
+
+const validation = ({id, value, isValidInfos, setIsValidInfos} : validationProps) => {
```

Comment:
```suggestion
const validateFormInfos = ({id, value, isValidInfos, setIsValidInfos} : validationProps) => {
```

함수명은 동사 혹은 동사+목적어로 짓는 편이 좋아요. 제 코드에 신경쓰지 못 한 부분이 있어서.. 혹시나 잘못된 제 네이밍을 참고하셨을까봐..🥲